### PR TITLE
Add bsdmainutils tool to Dockerfile to enable integration test on arm64

### PR DESCRIPTION
**Pull request by jongwu**
Build image for integration test on arm64 will fail for lack of
hexdump. Add bsdmainutils tool to eliminate that failure and let
build image succussfully
help me check it  @redbaron @mumoshu  @runcom @mrunalp  
Signed-off-by: Jianyong Wu <jianyong.wu@arm.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by jongwu** [comment]
I have signed for CLA, please check it.

**Comment by ashcrow** [comment]
LGTM

@rhatdan WDYT?

**Comment by runcom** [comment]
Looks good but need to sign the CLA

**Comment by runcom** [comment]
Well, amend and push force to re kick the CLA validation

**Comment by jongwu** [comment]
OK，let me try again @runcom

**Comment by jongwu** [comment]
now It's ok @runcom

**Comment by TomSweeneyRedHat** [comment]
LGTM and all happy green buttons. Thanks @jongwu!
